packages/react-docs/src/components/doc_reference.tsx

```diff
@@ -311,6 +215,15 @@ export class Documentation extends React.Component<DocumentationProps, Documenta
                             <div>{typeDefs}</div>
                         </div>
                     )}
+                <div
+                    style={{
+                        width: '100%',
+                        height: 1,
+                        backgroundColor: colors.grey300,
+                        marginTop: 32,
+                        marginBottom: 12,
+                    }}
+                />
```

Comment:
This would require moving `Container` and `Text` to `@0xproject/react-shared` since this file is in `@0xproject/react-docs`. I am planning on moving both these packages back into website in a future PR so will punt on this for now.